frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -398,6 +395,32 @@ function ReaderHighlight:addToMainMenu(menu_items)
     menu_items.long_press = {
         text = _("Long-press on text"),
         sub_item_table = {
+            {
+                text = _("Highlight Long-press interval"),
+                keep_menu_open = true,
+                callback = function()
+                    local SpinWidget = require("ui/widget/spinwidget")
+                    local items = SpinWidget:new{
+                        title_text = _("Highlight Long-press interval"),
+                        info_text = _([[
+If a touch is not released in this interval, it is considered a long-press. On document text, single word selection will not be triggered.
+
+The interval value is in seconds and can range from 3 seconds to 20 seconds.]]),
```

Comment:
I let @Frenzie review the wording and capitalization.

Reply:
Was this (more or less) copied from somewhere else? Because if not, there's a chance it should be.

Reply:
Btw, weren't we calling a "long press" something like "press and hold" now?

Reply:
It is copy from here and small modifications: https://github.com/koreader/koreader/blob/23d12c3f04a2d1135c0eb76f4678b7017a78f7ec/plugins/gestures.koplugin/main.lua#L574